source/tyr/tyr/resources.py

```diff
@@ -63,26 +63,41 @@
     cosmogony2cities,
     COSMOGONY_REGEXP,
 )
+from tyr import api
 from tyr.helper import get_instance_logger, save_in_tmp
 from tyr.fields import *
 from werkzeug.exceptions import BadRequest
-
 import werkzeug
 
-__ALL__ = ['Api', 'Instance', 'User', 'Key']
-
 
 class Api(flask_restful.Resource):
     def __init__(self):
         pass
 
     def get(self):
-        return marshal(models.Api.query.all(), api_fields)
+        def check_db():
+            tyr_db = sqlalchemy.create_engine(current_app.config['SQLALCHEMY_DATABASE_URI'])
+            try:
+                tyr_db.connect()
+                result = tyr_db.execute("SELECT version_num FROM alembic_version")
+                for row in result:
+                    return row['version_num']
+            except Exception as e:
+                logging.exception("Tyr db not reachable : {}".format(e.message))
+                return None
+
+        return {'db version': check_db(), 'api': marshal(models.Api.query.all(), api_fields)}
```

Comment:
this isn't related, I would have put the db version in a `/status`

Reply:
When I suggested to put it in /api in Slack, you reacted with thumbs up, as there's currently no endpoint /status.
If it is to be created, I suggest to do it in an other PR